packages/cli/generators/model/index.js

```diff
@@ -466,9 +469,11 @@ module.exports = class ModelGenerator extends ArtifactGenerator {
 
     debug('scaffolding');
 
-    Object.entries(this.artifactInfo.properties).forEach(([k, v]) =>
-      modelDiscoverer.sanitizeProperty(v),
-    );
+    Object.entries(this.artifactInfo.properties).forEach(([k, v]) => {
+      const builtinType = findBuiltinType(v.type);
+      if (builtinType) v.type = builtinType;
```

Comment:
Will `v.type` always be either a built in type by default or be found by `findBuildinType`? If it's not, can we put a warning?

Reply:
I think if `type` is not a built-in type then `findBuiltinType(v.type)` will returns undefined.

Reply:
`v.type` can be either a built-in type (e.g. `String` or `string`) or a user-defined type (e.g. `Address`).

In the case of built-in types, we want to convert the type name to all-lower-case (e.g. `string`), that's what these two lines L473-L474 are doing.

In the case of user-defined types, we keep the type name as-is.